calc/src/calc_fee.rs

```diff
@@ -0,0 +1,203 @@
+use crate::debug;
```

Comment:
I think you should have used git mv somewhere to avoid this. These are not new, right?